### PR TITLE
DOCS-2382: Add known errors

### PR DESCRIPTION
**Pull request by FrancescaNCook**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/payment-methods/billing-suite/klarna/user-guide/supported-addresses.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title : "Supported addresses"
 meta_title: "Klarna - Supported addresses - MultiSafepay Docs"
 meta_description: "Sign up. Build and test your payments integration. Explore our products and services. Use our API Reference, SDKs, and wrappers. Get support."
 read_more: "."
-weight: 8
+weight: 9
```
Comment (FrancescaNCook) [inline content/payment-methods/billing-suite/klarna/user-guide/supported-addresses.md:6]:
```suggestion
weight: 9
```

File: content/payment-methods/billing-suite/klarna/user-guide/changing-order-status-to-shipped.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
 ---
-title : "Changing order status from Completed to Shipped"
-meta_title: "Klarna - Changing order status from Completed to Shipped - MultiSafepay Docs"
+title : "Changing order status to Shipped"
```
Comment (FrancescaNCook) [inline content/payment-methods/billing-suite/klarna/user-guide/changing-order-status-to-shipped.md:2]:
```suggestion
title : "Changing order status to Shipped"
```

File: content/payment-methods/billing-suite/klarna/user-guide/changing-order-status-to-shipped.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+meta_title: "Klarna - Changing order status to Shipped - MultiSafepay Docs"
```
Comment (FrancescaNCook) [inline content/payment-methods/billing-suite/klarna/user-guide/changing-order-status-to-shipped.md:3]:
```suggestion
meta_title: "Klarna - Changing order status to Shipped - MultiSafepay Docs"
```

File: content/payment-methods/billing-suite/klarna/user-guide/changing-order-status-to-shipped.md [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
 meta_description: "Sign up. Build and test your payments integration. Explore our products and services. Use our API Reference, SDKs, and wrappers. Get support."
 read_more: "."
-weight: 3
+weight: 1
```
Comment (FrancescaNCook) [inline content/payment-methods/billing-suite/klarna/user-guide/changing-order-status-to-shipped.md:6]:
```suggestion
weight: 1
```

File: content/payment-methods/billing-suite/klarna/user-guide/changing-order-status-to-shipped.md [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
 ---
 
 Klarna does not support auto-ship functionality, so after you ship the order to the customer, you need to change the order status from **Completed** to **Shipped**. This prevents the order expiring, and lets Klarna initiate the billing process with the customer and pay the transaction out to your MultiSafepay balance. You can do this:
 
-- Automatically using our [API](https://docs.multisafepay.com/api/#update-an-order), which updates your MultiSafepay Control 
+- Automatically using our API, which updates your MultiSafepay Control. Make a PATCH `/orders` request.  See API Reference – [Update an order](https://docs.multisafepay.com/api/#update-an-order).
```
Comment (FrancescaNCook) [inline content/payment-methods/billing-suite/klarna/user-guide/changing-order-status-to-shipped.md:11]:
```suggestion
- Automatically using our API, which updates your MultiSafepay Control. Make a PATCH `/orders` request.  See API Reference – [Update an order](https://docs.multisafepay.com/api/#update-an-order).
```

File: content/payment-methods/billing-suite/klarna/user-guide/customizing-invoices.md [hunk 6]
```diff
@@ -3,7 +3,7 @@ title : "Customizing invoices"
 meta_title: "Klarna - Customizing invoices - MultiSafepay Docs"
 meta_description: "Sign up. Build and test your payments integration. Explore our products and services. Use our API Reference, SDKs, and wrappers. Get support."
 read_more: "."
-weight: 2
+weight: 4
```
Comment (FrancescaNCook) [inline content/payment-methods/billing-suite/klarna/user-guide/customizing-invoices.md:6]:
```suggestion
weight: 4
```

File: content/payment-methods/billing-suite/klarna/user-guide/handling-errors.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+---
+title : "Handling errors"
+meta_title: "Klarna - Handling errors - MultiSafepay Docs"
+meta_description: "Sign up. Build and test your payments integration. Explore our products and services. Use our API Reference, SDKs, and wrappers. Get support."
+read_more: "."
+weight: 10
```
Comment (FrancescaNCook) [inline content/payment-methods/billing-suite/klarna/user-guide/handling-errors.md:6]:
```suggestion
weight: 10
```

File: content/payment-methods/billing-suite/klarna/user-guide/placing-collection-period-on-hold.md [hunk 8]
```diff
@@ -3,7 +3,7 @@ title : "Pauzing the collection period"
 meta_title: "Klarna - Pauzing the collection period - MultiSafepay Docs"
 meta_description: "Sign up. Build and test your payments integration. Explore our products and services. Use our API Reference, SDKs, and wrappers. Get support."
 read_more: "."
-weight: 7
+weight: 6
```
Comment (FrancescaNCook) [inline content/payment-methods/billing-suite/klarna/user-guide/placing-collection-period-on-hold.md:6]:
```suggestion
weight: 6
```

File: content/payment-methods/billing-suite/klarna/user-guide/processing-refunds.md [hunk 9]
```diff
@@ -3,12 +3,15 @@ title : "Processing refunds"
 meta_title: "Klarna - Processing refunds - MultiSafepay Docs"
 meta_description: "Sign up. Build and test your payments integration. Explore our products and services. Use our API Reference, SDKs, and wrappers. Get support."
 read_more: "."
-weight: 1
+weight: 2
```
Comment (FrancescaNCook) [inline content/payment-methods/billing-suite/klarna/user-guide/processing-refunds.md:6]:
```suggestion
weight: 2
```

File: content/payment-methods/billing-suite/klarna/user-guide/processing-refunds.md [hunk 10]
```diff
@@ -3,12 +3,15 @@ title : "Processing refunds"
...
 ---
 
-You cannot refund more than the amount of the original transaction.
+- You cannot refund more than the amount of the original transaction.
```
Comment (FrancescaNCook) [inline content/payment-methods/billing-suite/klarna/user-guide/processing-refunds.md:9]:
```suggestion
- You cannot refund more than the amount of the original transaction.
```

File: content/payment-methods/billing-suite/klarna/user-guide/processing-refunds.md [hunk 11]
```diff
@@ -3,12 +3,15 @@ title : "Processing refunds"
...
 
-There is no time limit on refunding successful transactions, so long as the receiving bank can process the refund.
+- There is no time limit on refunding successful transactions, so long as the receiving bank can process the refund.
```
Comment (FrancescaNCook) [inline content/payment-methods/billing-suite/klarna/user-guide/processing-refunds.md:11]:
```suggestion
- There is no time limit on refunding successful transactions, so long as the receiving bank can process the refund.
```

File: content/payment-methods/billing-suite/klarna/user-guide/processing-refunds.md [hunk 12]
```diff
@@ -3,12 +3,15 @@ title : "Processing refunds"
...
+
+## Via our API
```
Comment (FrancescaNCook) [inline content/payment-methods/billing-suite/klarna/user-guide/processing-refunds.md:13]:
```suggestion
## Via our API
```

File: content/payment-methods/billing-suite/klarna/user-guide/processing-refunds.md [hunk 13]
```diff
@@ -3,12 +3,15 @@ title : "Processing refunds"
...
+Make a POST `/orders/{order_id}/refunds` request. See API Reference – [Refund with shopping cart](/api/#refund-with-shopping-cart).
```
Comment (FrancescaNCook) [inline content/payment-methods/billing-suite/klarna/user-guide/processing-refunds.md:14]:
```suggestion
Make a POST `/orders/{order_id}/refunds` request. See API Reference – [Refund with shopping cart](/api/#refund-with-shopping-cart).
```

File: content/payment-methods/billing-suite/klarna/user-guide/reconciliation-and-reports.md [hunk 14]
```diff
@@ -3,7 +3,7 @@ title : "Reconciliation and reports"
 meta_title: "Klarna - Reconciliation and reports - MultiSafepay Docs"
 meta_description: "Sign up. Build and test your payments integration. Explore our products and services. Use our API Reference, SDKs, and wrappers. Get support."
 read_more: "."
-weight: 6
+weight: 7
```
Comment (FrancescaNCook) [inline content/payment-methods/billing-suite/klarna/user-guide/reconciliation-and-reports.md:6]:
```suggestion
weight: 7
```

File: content/payment-methods/billing-suite/klarna/user-guide/reservation-and-invoice-number.md [hunk 15]
```diff
@@ -3,7 +3,7 @@ title : "Reservation and invoice numbers"
 meta_title: "Klarna - Reservation and invoice number - MultiSafepay Docs"
 meta_description: "Sign up. Build and test your payments integration. Explore our products and services. Use our API Reference, SDKs, and wrappers. Get support."
 read_more: "."
-weight: 10
+weight: 8
```
Comment (FrancescaNCook) [inline content/payment-methods/billing-suite/klarna/user-guide/reservation-and-invoice-number.md:6]:
```suggestion
weight: 8
```

File: content/payment-methods/billing-suite/klarna/user-guide/handling-errors.md [hunk 16]
```diff
@@ -0,0 +1,9 @@
...
+---
+
+If you receive a `code:BAD_VALUE, Bad value: order_lines[0].reference` error from Klarna, try using shorter SKU numbers (e.g. fewer than 9 characters).
```
Comment (Kris-MultiSafepay) [inline content/payment-methods/billing-suite/klarna/user-guide/handling-errors.md:9]: As a merchant, I'd like to know where I would encounter this error and where I can alter the SKU number. 

The error seems to be displayed on the MultiSafepay payment pages, which means that a merchant has to test Klarna payments to see the error (or he/she will find out through customers).

Providing instructions for altering the SKU is a little more tricky as we don't mention SKUs anywhere in our API reference. I suppose that some of the platforms for which we have ecommerce integrations use SKUs, which are provided as values in Klarna order requests. When merchants are building a custom integration, I assume the SKUs would be provided as values for `shopping_cart.items.merchant_item_id`.

**Review by Kris-MultiSafepay — CHANGES_REQUESTED** [review]
See comment

File: content/payment-methods/billing-suite/klarna/user-guide/handling-errors.md (hunk 16 above)
Comment (FrancescaNCook) [reply content/payment-methods/billing-suite/klarna/user-guide/handling-errors.md:9]: You're spot on, but I think this is a bit in depth for now. Robert said only one merchant has brought it up and he wasn't even sure it was worth mentioning on Docs. I think the hint is probably enough at the mo?

**Pull request merged**